README.md

```diff
@@ -409,56 +409,57 @@ Good starting values:
 
       enabled: true
       items:
-          cobblestone: 300
-          netherrack: 300
-          sand: 300
-          red_sand: 300
-          gravel: 300
-          dirt: 300
-          grass: 300
-          pumpkin: 300
-          melon_slice: 300
-          kelp: 300
-          bamboo: 300
-          sugar_cane: 300
-          twisting_vines: 300
-          weeping_vines: 300
-          oak_leaves: 300
-          spruce_leaves: 300
-          birch_leaves: 300
-          jungle_leaves: 300
-          acacia_leaves: 300
-          dark_oak_leaves: 300
-          cactus: 300
-          diorite: 300
-          granite: 300
-          andesite: 300
-          scaffolding: 600
+        cobblestone: 300
+        netherrack: 300
+        sand: 300
+        red_sand: 300
+        gravel: 300
+        dirt: 300
+        grass: 300
+        pumpkin: 300
+        melon_slice: 300
+        kelp: 300
+        bamboo: 300
+        sugar_cane: 300
+        twisting_vines: 300
+        weeping_vines: 300
+        oak_leaves: 300
+        spruce_leaves: 300
+        birch_leaves: 300
+        jungle_leaves: 300
+        acacia_leaves: 300
+        dark_oak_leaves: 300
+        mangrove_leaves: 300
+        cactus: 300
+        diorite: 300
+        granite: 300
+        andesite: 300
+        scaffolding: 600
 ```
 
 This list lets you set alternative time (in ticks) to despawn certain types of dropped items faster or slower than default. This option can be used instead of item clearing plugins along with `merge-radius` to improve performance.
 
 #### redstone-implementation
 
-`Good starting value: alternate-current`
+`Good starting value: ALTERNATE_CURRENT`
```

Comment:
afaik lowercase works here, no need to change this to uppercase

Reply:
Yes, this is not important.
`alternate-current` is automatically changed to `ALTERNATE_CURRENT`.